.github/workflows/basic.yml

```diff
@@ -119,6 +122,10 @@ jobs:
             compiler: {c: gcc, cxx: g++}
             shared_library: 'OFF'
             os_provider: 'OFF'
+    env:
+      BUILD_DIR : "${{github.workspace}}/build/"
+      INSTL_DIR : "${{github.workspace}}/build/install-dir"
+      SHARED_LIBRARY: ${{ matrix.shared_library == 'ON' && '--shared-library' || ''}}        
```

Comment:
I don't understand this combination of && and || '' - could you add a comment here?

Reply:
Added some comments

Reply:
thanks, now I get it